helix-view/src/editor.rs

```diff
@@ -229,6 +229,113 @@ where
     Ok(chars)
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct CompletionItemKindWrapper(pub CompletionItemKind);
```

Comment:
Why a wrapper?

Reply:
Removed the wrapper in https://github.com/helix-editor/helix/pull/9996/commits/3790060e29afc6500445029a3b81a7c748ba6d07, looks good now?